### PR TITLE
Include MC for Run3

### PR DESCRIPTION
**Pull request by MataiasCost**
The file outputForTwik.twiki, are the table os MC from run3, he is made using the code "runCMSDrivers_mc2022.sh"

### CODE REVIEW

File: ConditionsConsumed/runCMSDrivers_mc2018.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 echo "------------------------------"
 echo "Running step-1/9: GEN"
 echo "------------------------------"
-cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions 105X_upgrade2018_realistic_v6 -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
+cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions auto:phase1_2022_realistic -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2018.sh:5]: I think this should be kept as `--conditions 105X_upgrade2018_realistic_v6`

File: ConditionsConsumed/runCMSDrivers_mc2022.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+echo "------------------------------"
+echo "Running step-2/7: L1"
+echo "------------------------------"
+cmsDriver.py step2  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s L1 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step3_DIGI.root --fileout output_step4_L1.root --customise_commands="process.load('Configuration.StandardSequences.Digi_cff') \n process.GlobalTag.DumpStat =cms.untracked.bool(True)" --outputCommands "keep *" |& tee output_step4_L1.log
+
+
+
+#4.1: L1+DIGI
+cmsDriver.py step3  --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s DIGI:pdigi_valid,L1 --datatier GEN-SIM-DIGI-RAW --eventcontent FEVTDEBUGHLT --filein file:output_step2_SIM.root --fileout output_step3_DIGI_L1.root  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' --outputCommands 'keep *_simCastorDigis_*_*' |& tee output_step3_DIGI_l1.log
+
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2022.sh:23]: Which of these two steps will get picked up by the script `getAlCaCondInGT.py` ? I think you should keep only the cmsDriver commands that produce the output file that will be used by the script.

File: ConditionsConsumed/runCMSDrivers_mc2022.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+cmsDriver.py step8 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s PAT --datatier MINIAODSIM --eventcontent MINIAODSIM --filein file:output_step7_AODSIM.root --fileout output_step8_MINIAODSIM.root --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)' |& tee output_step8_MINIAODSIM.log
+
+
+
+#9 NANOAODSIM
+echo "------------------------------"
+echo "Running step-7/7: NANOAODSIM"
+echo "------------------------------"
+# cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step7_AODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
+cmsDriver.py step5 --runUnscheduled --conditions auto:phase1_2022_realistic -n 5 --era Run3 -s RAW2DIGI,L1Reco,RECO,RECOSIM,PAT,NANO --datatier NANOAODSIM --eventcontent NANOAODSIM --filein file:output_step8_MINIAODSIM.root --fileout output_step9_NANOAODSIM.root --customise_commands='process.GlobalTag.DumpStat=cms.untracked.bool(True)' |& tee output_step9_NANOAODSIM.log
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2022.sh:70]: Is there a way to run the NANO step separately?

**Comment by malbouis** [comment]
@MataiasCost, I think it would be also good to update the script `getAlCaCondInGT.py` accordingly, taking into account the steps that you have merged when you write the table.

**Comment by malbouis** [comment]
It would also be nice to include the link of the [2022 twiki](https://twiki.cern.ch/twiki/bin/viewauth/CMS/AlCaDBHLT2022) somewhere in here, like in the README for example.

File: ConditionsConsumed/runCMSDrivers_mc2018.sh (hunk 1 above)
Comment (MataiasCost) [reply ConditionsConsumed/runCMSDrivers_mc2018.sh:5]: You are right, i change this file (just to make some tests), and forgot to change again. I will fix that

File: ConditionsConsumed/runCMSDrivers_mc2018.sh [hunk 4]
```diff
@@ -2,8 +2,7 @@
 echo "------------------------------"
 echo "Running step-1/9: GEN"
 echo "------------------------------"
-cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions 105X_upgrade2018_realistic_v6 -n 5 --era Run2_2018 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
-
+cmsDriver.py TTbar_13TeV_TuneCUETP8M1_cfi  --conditions auto:run2_mc -n 5 --era Run2_2016 -s GEN --fileout output_step1_GEN.root --beamspot Realistic50ns13TeVCollision  --customise_commands='process.GlobalTag.DumpStat =cms.untracked.bool(True)'|& tee output_step1_GEN.log
```
Comment (malbouis) [inline ConditionsConsumed/runCMSDrivers_mc2018.sh:5]: --era Run2_2018

File: ConditionsConsumed/runCMSDrivers_mc2022.sh (hunk 3 above)
Comment (tvami) [reply ConditionsConsumed/runCMSDrivers_mc2022.sh:70]: Yes we should definitely run this step by step

**Pull request closed without merging**